### ISSUE TITLE
Features are not announced (by NVDA, at least) when `<map is="web-map" ...>` is used

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#472]
For [example](https://maps4html.org/experiments/custom-projections/bc/).

@Malvoz I know you have mentioned this issue before, perhaps it is the major accessibility issue with the custom built-in `<map>`.  Can you link to where you've discussed this before, or explain again.  Thanks for your patience.

**Comment by Malvoz** [comment]
Yes that's correct. The HTML AAM issue is: https://github.com/w3c/html-aam/issues/292.

It's mostly (or only) a chromium bug, which Scott O'Hara filed: https://bugs.chromium.org/p/chromium/issues/detail?id=1208405.

**Comment by prushforth** [comment]
Firefox wins again

**Comment by Malvoz** [comment]
BTW we've mentioned the issue in:
- https://github.com/Maps4HTML/Web-Map-Custom-Element#a-parellel-customized-built-in-element
- https://github.com/Maps4HTML/MapML-Proposal/blob/master/high-level-api.md#polyfill

And yeah, that Chromium bug is _really_ bad. The `<map>` polyfill will _always_ have this accessibility issue in all the versions before a fix lands.

**Comment by prushforth** [comment]
Hopefully the evergreen browser trick will make the problem [evaporate](https://en.wikipedia.org/wiki/Hawking_radiation) eventually.

**Comment by Malvoz** [comment]
There's no action to be taken here, so I think we should close this.

Although we should not be endorsing the use of `<map is="web-map">`. I'm in favor of not including it in the web map docs (i.e. removing https://maps4html.org/web-map-doc/docs/maps/web-map, perhaps moving that content elsewhere, such as the experiments repo or here). I think that web page may also be a bit confusing/distracting to developers.

**Comment by Malvoz** [comment]
> i.e. removing https://maps4html.org/web-map-doc/docs/maps/web-map

Another (perhaps temporary) option is to keep the resource at https://maps4html.org/web-map-doc/docs/maps/web-map but exclude the <q>`<map is="web-map">`</q> link from the <q>Mapping elements</q> menu item in the navigation menu.

**Comment by Malvoz** [comment]
I'll go ahead and close this, as it is an issue due to a browser bug.
